ikasaneip/ootb/module/scheduler-agent/jar/src/main/java/org/ikasan/ootb/scheduler/agent/module/model/EnrichedContextualisedScheduledProcessEvent.java

```diff
@@ -77,8 +79,21 @@ public List<ContextParameterInstance> getContextParameters() {
     @Override
     public String toString() {
         return "EnrichedContextualisedScheduledProcessEvent{" +
+            // this
             ", detachableProcess=" + detachableProcess +
             ", contextParameters=" + contextParameters +
+
+            // ContextualisedScheduledProcessEventImpl
+            ", contextId=" + getContextName() +
+            ", childContextIds=" + getChildContextNames() +
+            ", contextInstanceId=" + getContextInstanceId() +
+            ", skipped=" + isSkipped() +
+            // getInternalEventDrivenJob() prevents further logging so is suppressed.
+            //", internalEventDrivenJob=" + getInternalEventDrivenJob() +
```

Comment:
@davidhilton68 please remove commented out line if no longer necessary

Reply:
Good call, updates to remove the commented out code but kept the comment to stop anyone falling into same issue.